scripts/variantstore/wdl/GvsQuickstartIntegration.wdl

```diff
@@ -0,0 +1,224 @@
+        "gs://fc-2b4456d7-974b-4b67-90f8-63c2fd2c03d4/gvcfs/HG00429.haplotypeCalls.er.raw.vcf.gz.vcf.gz",
+        "gs://fc-2b4456d7-974b-4b67-90f8-63c2fd2c03d4/gvcfs/HG00444.haplotypeCalls.er.raw.vcf.gz.vcf.gz",
+        "gs://fc-2b4456d7-974b-4b67-90f8-63c2fd2c03d4/gvcfs/HG00447.haplotypeCalls.er.raw.vcf.gz.vcf.gz",
+        "gs://fc-2b4456d7-974b-4b67-90f8-63c2fd2c03d4/gvcfs/HG00450.haplotypeCalls.er.raw.vcf.gz.vcf.gz"
+    ]
+
+    Array[File] input_vcf_indexes = [
+        "gs://fc-2b4456d7-974b-4b67-90f8-63c2fd2c03d4/gvcfs/HG00405.haplotypeCalls.er.raw.vcf.gz.vcf.gz.tbi",
+        "gs://fc-2b4456d7-974b-4b67-90f8-63c2fd2c03d4/gvcfs/HG00408.haplotypeCalls.er.raw.vcf.gz.vcf.gz.tbi",
+        "gs://fc-2b4456d7-974b-4b67-90f8-63c2fd2c03d4/gvcfs/HG00418.haplotypeCalls.er.raw.vcf.gz.vcf.gz.tbi",
+        "gs://fc-2b4456d7-974b-4b67-90f8-63c2fd2c03d4/gvcfs/HG00420.haplotypeCalls.er.raw.vcf.gz.vcf.gz.tbi",
+        "gs://fc-2b4456d7-974b-4b67-90f8-63c2fd2c03d4/gvcfs/HG00423.haplotypeCalls.er.raw.vcf.gz.vcf.gz.tbi",
+        "gs://fc-2b4456d7-974b-4b67-90f8-63c2fd2c03d4/gvcfs/HG00427.haplotypeCalls.er.raw.vcf.gz.vcf.gz.tbi",
+        "gs://fc-2b4456d7-974b-4b67-90f8-63c2fd2c03d4/gvcfs/HG00429.haplotypeCalls.er.raw.vcf.gz.vcf.gz.tbi",
+        "gs://fc-2b4456d7-974b-4b67-90f8-63c2fd2c03d4/gvcfs/HG00444.haplotypeCalls.er.raw.vcf.gz.vcf.gz.tbi",
+        "gs://fc-2b4456d7-974b-4b67-90f8-63c2fd2c03d4/gvcfs/HG00447.haplotypeCalls.er.raw.vcf.gz.vcf.gz.tbi",
+        "gs://fc-2b4456d7-974b-4b67-90f8-63c2fd2c03d4/gvcfs/HG00450.haplotypeCalls.er.raw.vcf.gz.vcf.gz.tbi",
+    ]
+
+    call Prepare {
+        input:
+            branch_name = branch_name
+    }
+
+    call GvsUnified.GvsUnified {
+        input:
+            dataset_name = Prepare.dataset_name,
+            project_id = "spec-ops-aou",
+            external_sample_names = external_sample_names,
+            gatk_override = Prepare.jar,
+            input_vcfs = input_vcfs,
+            input_vcf_indexes = input_vcf_indexes,
+            filter_set_name = "quickit",
+            create_filter_set_scatter_count = 20,
+            extract_table_prefix = "quickit",
+            extract_scatter_count = 100,
+            # Force filtering off as it is not deterministic and the initial version of this integration test does not
+            # allow for inexact matching of actual and expected results.
+            extract_do_not_filter_override = true
+    }
+
+    call AssertIdenticalOutputs {
+        input:
+            expected_output_prefix = expected_output_prefix,
+            actual_vcfs = GvsUnified.output_vcfs
+    }
+
+    output {
+        Array[File] output_vcfs = GvsUnified.output_vcfs
+        Array[File] output_vcf_indexes = GvsUnified.output_vcf_indexes
+        Float total_vcfs_size_mb = GvsUnified.total_vcfs_size_mb
+        File manifest = GvsUnified.manifest
+        Boolean done = true
+    }
+}
+
+task Prepare {
```

Comment:
nit: can this be something else, like "setup", just to not create any confusion with the `PrepareRangesCallset` step?